CHANGELOG

```diff
@@ -11,6 +11,8 @@ Version 1.1.23 under development
 - Bug #4306: Add PHP 8 support (samdark)
 - Enh #4308: PHP 7.3 compatibility: Add `samesite` as a session cookie option (tomfotherby)
 - Bug #4310: Items on memcache won't expire due to memcache difference in internal clock (nikolasr200)
+- Chg: Exceptions thrown while loading fixtures now contain details about the error location (BBoom)
+- Chg: Missing fixture files now throw exceptions (BBoom)
```

Comment:
```suggestion
- Chg #4314: Exceptions thrown while loading fixtures now contain details about the error location (BBoom)
- Chg #4314: Missing fixture files now throw exceptions (BBoom)
```

Reply:
"Missing fixture files now throw exceptions" could be a problem. As far as I understand, previously they weren't loaded at all, right?

Reply:
That is correct, previously they were discarded silently. I would consider that a bug, and regard this change as a bugfix instead of "normal behavior" and a BC-break. What do you think?

Reply:
OK. That makes sense,.